openthread/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+include(${NRFXLIB_DIR}/common.cmake)
+nrfxlib_calculate_lib_path(lib_path)
+set(OT_LIB_PATH ${CMAKE_CURRENT_SOURCE_DIR}/${lib_path})
+
+if (NOT CONFIG_OPENTHREAD_BUILD_SOURCES)
+  if(NOT EXISTS ${OT_LIB_PATH})
+    message(WARNING "This platform is not certified by OpenThread yet.
```

Comment:
Shouldn't it be rather something like: "Missing OT library path, fix it or compile OT from source"?

Reply:
${OT_LIB_PATH} is set automatically by NCS build scripts to path pointing to the correct platform (e.g lib/cortex-m4/soft-float for nRF52840). All libraries shall be provided by Nordic.
The idea here is that user does not update those libraries. If the library is not available this means this platform is not certified or is not supported(making it not certified as well)